docs/css/style.css

```diff
@@ -96,10 +102,15 @@ html, body {
   position: relative;
   top: -10vh;
   font-size: 1em;
-  font-weight: thin;
+  font-weight: lighter;
 }
 .videoOverlay {
-	font-size: 2em
+  font-size: 2em;
+  cursor: pointer;
+  transition: .1s ease-in-out;
+}
+.videoOverlay:hover {
+  opacity: .8;
```

Comment:
Why are you setting the opacity to 0.8, it will make the video transparent?

Reply:
Sorry, I just want to change the button opacity, modify to:
```css
.videoOverlay:hover .toggleVideo {
  opacity: .8;
}
```